packages/integrations/package.json

```diff
@@ -16,13 +16,15 @@
   "module": "esm/index.js",
   "types": "dist/index.d.ts",
   "dependencies": {
-    "@sentry/types": "5.22.3",
-    "@sentry/utils": "5.22.3",
+    "@sentry/browser": "^5.22.3",
```

Comment:
Remove dependency

Reply:
removed  "@sentry/browser": "^5.22.3", in new Pull request.